brayns/engine/Model.h

```diff
@@ -387,7 +388,7 @@ class Model
      * existing geometry in the model. Missing materials are created with the
      * default parameters
      */
-    BRAYNS_API void createMissingMaterials();
+    BRAYNS_API void createMissingMaterials(const PropertyMap& properties = {});
```

Comment:
that is not called other than this anyways, so the parameter is not needed for this function?!

Reply:
Properties are forwarded to the material itself (line 558). It's true that Brayns never calls this method with properties, but this can still be done by a plugin for instance (that's actually what I do and why I need this extra parameter).

Reply:
Shouldn't you pass the material type name as well?

Reply:
As discussed offline, this `createMissingMaterials()` shall be gone completely. The CircuitViewer plugin uses it though, but imo that shouldn't be necessary.  @hernando could you have a look there?

Reply:
I'll do.

Reply:
Done